index.rst

```diff
@@ -3,11 +3,6 @@
 Symfony Documentation
 =====================
 
-.. toctree::
-    :hidden:
-
-    changelog
```

Comment:
On Mon, Jan 26, 2015 at 12:18:22AM -0800, Wouter J wrote:
>    In [1]index.rst:
> 
>  > @@ -3,11 +3,6 @@
>  >  Symfony Documentation
>  >  =====================
>  >
>  > -.. toctree::
>  > -    :hidden:
>  > -
>  > -    changelog
> 
>    it isn't now, as symfony.com builds use another conf.py setting.
>    Otherwise, we can always use
>    [2]http://stackoverflow.com/questions/13529985/how-to-set-sphinxs-exclude-patterns-from-the-command-line
> 
>    —
>    Reply to this email directly or [3]view it on GitHub.
> 
>    Reverse link: [4]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/symfony/symfony-docs/pull/4917#discussion_r23516318
>    2. http://stackoverflow.com/questions/13529985/how-to-set-sphinxs-exclude-patterns-from-the-command-line
>    3. https://github.com/symfony/symfony-docs/pull/4917/files#r23516318
>    4. https://github.com/symfony/symfony-docs/pull/4917/files#r23516318

Ah good find. Although I guess from what you say we can leave it as is.